changelog.md

```diff
@@ -1,7 +1,12 @@
 # Backpack changelog
 
 ## UNRELEASED
-_Nothing Yet_
+### 2017-10-16 - React Lists and List Items can now have a className attribute added
+
+**Added:**
+- bpk-component-list
+  - The `className` attribute has been added to BpkList and BpkListItem, mirroring functionality from other props that implement it
```

Comment:
Can we call props "props" instead of attributes to be consistent with React and the rest of the codebase? I believe the second part should say "mirroring functionallity in other components."

Reply:
Not sure why I said attributes there, I wasn't sure if 'attributes' or 'props' was the correct terminology

Reply:
There we are, fixed